examples/basic/app.js

```diff
@@ -73,6 +75,7 @@ const vueInstance = new Vue({
           </li>
         </router-link>
         <li><router-link to="/foo" replace>/foo (replace)</router-link></li>
+        <li><router-link to="/query/A%">/query/A%</router-link></li>
```

Comment:
This has to be 

```suggestion
        <li><router-link to="/query/A%">/query/A%25</router-link></li>
```

in order to work in all browsers and when copying the URL into a new window to any browser after clicking the link